### ISSUE TITLE
Css is not correct

### ISSUE THREAD

**Issue by reinvanleirsberghe** [nxtend-team/nxtend#404]
After generating a project and serving is with `nx serve my-ionic-app`, I get underneath.

I have the feeling that some CSS is not rendered correctly... Any ideas?

![image](https://user-images.githubusercontent.com/4301574/104807420-4280e880-57df-11eb-9968-6f99077885a9.png)

**Comment by devinshoemaker** [comment]
I don't think this is an issue with the CSS, but it does look like one of the templates may have that text at the bottom in the wrong place. It should be above the text that's centered. I'll take a look at this soon. You should be able to continue developing your app without issues.
